hack/test-pre-release-ossm-images.sh

```diff
@@ -63,40 +123,41 @@ spec:
   sourceType: grpc
 EOF
 
-MAX_TRIES=60
+MAX_TRIES=100
 TRIES=0
 
 while (( TRIES < MAX_TRIES )); do
     CATALOG_STATE=$(oc get catalogsource -n openshift-marketplace custom-istio-catalog -ojson | jq -r '.status.connectionState.lastObservedState')
     if [[ "$CATALOG_STATE" = "READY" ]]; then
-        echo "Istio custom catalog source is ready!"
+        echo "> Istio custom catalog source is ready!"
         break
     fi
-    echo "($((TRIES+1))/${MAX_TRIES}) Istio catalog source is not ready, retrying..."
-    ((TRIES++))
+    TRIES=$((TRIES+1))
+    echo "(${TRIES}/${MAX_TRIES}) Istio catalog source is not ready, retrying..."
     sleep 2
 done
 
 if (( TRIES >= MAX_TRIES )); then
-    echo "Exceeded max tries waiting for catalog source to become ready"
+    echo "> Exceeded max tries waiting for catalog source to become ready"
     exit 3
 fi
 
 CUSTOM_CATALOG_SOURCE=custom-istio-catalog
 TRIES=0
 while (( TRIES < MAX_TRIES )); do
-    OSSM_VERSION=$(oc get packagemanifests -n openshift-marketplace -o jsonpath="{range .items[?(@.metadata.labels.catalog=='custom-istio-catalog')]} {.status.channels[*].currentCSV}{\"\n\"}{end}" | grep servicemeshoperator3 | awk '{print $NF}')
+    OSSM_VERSION=$(oc get packagemanifests -n openshift-marketplace -o json | jq -r '.items[] | select(.metadata.labels.catalog=="custom-istio-catalog") | .status.channels[] | select(.currentCSV | contains("servicemeshoperator3")) | .currentCSV')
     if [[ "$OSSM_VERSION" != "" ]]; then
-        echo "OSSM version found!"
+        echo "> OSSM version found!"
         break
     fi
-    echo "($((TRIES+1))/${MAX_TRIES}) OSSM version is not available, retrying..."
-    ((TRIES++))
+    TRIES=$((TRIES+1))
+    echo "(${TRIES}/${MAX_TRIES}) OSSM version is not available, retrying..."
     sleep 2
 done
 if (( TRIES >= MAX_TRIES )); then
-    echo "OSSM version is not found in $CUSTOM_CATALOG_SOURCE"
+    echo "> OSSM version is not found in $CUSTOM_CATALOG_SOURCE"
     exit 4
 fi
-echo "OSSM version from the custom catalog source: $OSSM_VERSION"
+OSSM_VERSION=$(oc get packagemanifests -n openshift-marketplace -o json | jq -r '.items[] | select(.metadata.labels.catalog=="custom-istio-catalog") | .status.channels[] | select(.currentCSV | contains("servicemeshoperator3")) | .currentCSV' | tail -n 1)
```

Comment:
Can't we reuse `OSSM_VERSION` which was already set in the polling loop above? Because the command looks very similar, it would be redundant to repeat it twice.